src/js/i2c.js

```diff
@@ -199,13 +193,14 @@ I2C.prototype.stream = function(cmd, len, delay) {
       if (err) {
         return _this.emit('error', err);
       } else {
-        return _this.emit('data', {
+        _this.emit('data', {
           address: _this.address,
           data: data,
           cmd: cmd,
           length: len,
           timestamp: Date.now()
         });
+        _this.stream(cmd, len, delay);
```

Comment:
Originally, this callback function returns boolean which says event emitter emits event successfully or not. And I think the correspondency with the past is broken.